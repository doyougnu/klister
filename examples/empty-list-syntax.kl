#lang kernel

(import "prim-syntax.kl")

(example
    (empty-list-syntax 'some-syntax))
