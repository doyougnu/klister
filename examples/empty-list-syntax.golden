#[empty-list-syntax.kl:6.25-6.36]<()> : Syntax
